packages/client/src/lib.js

```diff
@@ -109,12 +110,12 @@ class Web3Storage {
         })),
         blockstore,
         wrapWithDirectory,
-        maxChunkSize: 1048576,
+        maxChunkSize,
```

Comment:
Hold up, this shouldn't have been changed - this is the max block size (I know confusingly named) which we set to 1MB.

```suggestion
        maxChunkSize: 1048576,
```

Although we should really pull this out into a constant and use it here, so we don't get confused again!:

```js
const MAX_BLOCK_SIZE = 1048576
```

Reply:
@alanshaw Thanks for the suggestions, I already did some validation on the `maxChunkSize `and added the MAX_BLOCK_SIZE constant, have a look when you have time